code/game/machinery/vending_types.dm

```diff
@@ -921,7 +921,7 @@
 		/obj/item/reagent_containers/ladle = 4,
 		/obj/item/storage/toolbox/lunchbox/nt = 6,
 		/obj/item/reagent_containers/glass/rag = 8,
-		/obj/item/evidencebag/plasticbag = 6,
+		/obj/item/evidencebag/plastic_bag = 20,
```

Comment:
Why the path change?
First we have evidencebag and then we have plastic_bag.
Either change evidence_bag as well or restore plasticbag.
But mixing both in the same path is strange.

Reply:
Also removed the bugfix label as the path change is not a bug fix

Reply:
path restored.